demo/full/scripts/controllers/charts/BufferContent.tsx

```diff
@@ -1,7 +1,8 @@
 import * as React from "react";
 import BufferContentGraph from "../../components/BufferContentGraph";
 import useModuleState from "../../lib/useModuleState";
-import type { IBufferedData, IPlayerModule } from "../../modules/player/index";
+import type { IPlayerModule } from "../../modules/player/index";
+import type { IBufferedChunkSnapshot } from "../../../../../src/core/segment_sinks/segment_buffers_store";
```

Comment:
I don't know if I like going deep in the RxPlayer code in the demo here :/

Couldn't we re-define that type in the demo and let TypeScript check if they are equal?

Reply:
I have made the changes